contracts/Swapper.sol

```diff
@@ -50,30 +58,14 @@ contract Swapper {
             }
         }
 
-        idxFound = 0;
```

Comment:
Rather than adjusting the swapper, can you please make a `SwapperV2.sol` so that the V1 can still be used for lockers that may not have a zap?